cockatrice/src/game/player/player_actions.cpp

```diff
@@ -878,7 +878,8 @@ void PlayerActions::actCreateAnotherToken()
 
 void PlayerActions::setLastToken(CardInfoPtr cardInfo)
 {
-    if (cardInfo == nullptr || !player->getPlayerMenu()->getUtilityMenu()->createAnotherTokenActionExists()) {
+    if (cardInfo == nullptr || !player->getPlayerMenu()->getUtilityMenu() ||
```

Comment:
```suggestion
    if (cardInfo == nullptr || player->getPlayerMenu()->getUtilityMenu() == nullptr||
```
this is easier to read, more explicit